docs/make.jl

```diff
@@ -68,6 +68,8 @@ withenv("GKSwstype" => "nul") do
             "Home" => "index.md",
             "Introduction" => "intro.md",
             "API" => "api.md",
+            "Intro to Vectors" => "intro-to-vectors.md",
+            "Intro to SEM" => "intro-to-sem.md",
```

Comment:
I don't think people will generally find these docs unless there's a link in the navigation bar. (I happened to find it by searching in the code).

Reply:
Yes, but even if they are subsections they be in the nav bar, can't they? Like we see the different examples as soon as one clicks on the "Examples" section. I'd like to propose we make a Theoretical Framework, Mathematical Framework (or whatever we want to call it) section and put these  under that. Otherwise they are just random as is in the TOC and does not fit the whole

Reply:
That's fine, we can improve the organization in a separate PR--feel free to do so. I'd just like to make these pages more visible in the docs.